install/0000_90_cluster-version-operator_02_servicemonitor.yaml

```diff
@@ -64,6 +66,7 @@ spec:
       expr: |
         sum by (channel,upstream) (cluster_version_available_updates) > 0
       labels:
+        namespace: openshift-cluster-version
```

Comment:
`UpdateAvailable` already has `namespace` in Telemetry (this time via `cluster_version_available_updates`), although we probably lost that with 0721e7bfa (#643).  Can we add `namespace` to the `sum by` instead, with:

```
sum by (channel,namespace,upstream) (cluster_version_available_updates) > 0
```

and drop this line?

Reply:
Thank you, fixed in https://github.com/openshift/cluster-version-operator/pull/800/commits/d525eb97a2c971b4837210963d86d3b617452891.